common/src/main/java/com/mapbox/vision/common/BaseActivity.kt

```diff
@@ -0,0 +1,92 @@
+package com.mapbox.vision.common
+
+import android.content.pm.PackageManager
+import android.os.Build
+import android.os.Bundle
+import android.text.Html
+import android.view.WindowManager
+import android.widget.Toast
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.content.ContextCompat
+import com.mapbox.vision.mobile.core.utils.SystemInfoUtils
+import com.mapbox.vision.mobile.core.utils.snapdragon.SupportedSnapdragonBoards
+import com.mapbox.vision.utils.VisionLogger
+
+abstract class BaseActivity : AppCompatActivity() {
```

Comment:
BaseVisionActivity to easy understanding for external devs to understand it's our custom base activity?

Reply:
done